nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -407,9 +417,11 @@ define([
                 selectedFileType,
                 'Parent comm bus for parameters widget'
             );
-            paramBus.on('parameter-changed', (message) => {
-                updateModelParameterValue(selectedFileType, PARAM_TYPE, message);
-            });
+            if (!viewOnly) {
+                paramBus.on('parameter-changed', (message) => {
```

Comment:
When in view only mode, we don't change state. Ideally `parameter-changed` wouldn't be called, but there are a number of cases (most?) where parameter widgets trigger that message on startup. In case we miss any in future work, this should just prevent the message from doing unnecessary updates.